es-vue-base/src/lib-utils/nav-bar-global-content.js

```diff
@@ -306,15 +306,15 @@ export default (
                         },
                         {
                             name: 'Get personalized quotes',
-                            link: `${ES_DOMAIN}/market/start/`,
+                            link: `${ES_DOMAIN}/shop/home-solar/`,
```

Comment:
We still want this "Get personalized quotes" link to point to `/market/start/`, as that's meant to be a direct way for customers to start the signup process.

The "Home solar" link that is a parent of this one, which currently points to `/market-intro/`, is the one we want to change to point to `/shop/home-solar/`.